### PR TITLE
Update selected file label on the ide start up

### PR DESCRIPTION
**Pull request by mkondratek**
## Test plan
- runIde
- check if the label reflects the currently opened file on start-up (it can take a few seconds)

### CODE REVIEW

File: src/main/kotlin/com/sourcegraph/cody/context/EmbeddingStatusView.kt [hunk 1]
```diff
@@ -103,7 +103,7 @@ class EmbeddingStatusView(private val project: Project) : JPanel() {
   }
 
   fun setOpenedFileName(fileName: String, filePath: String?) {
-    openedFileContent.setText(fileName)
+    openedFileContent.text = fileName
     openedFileContent.setToolTipText(filePath)
```
Comment (pkukielka) [inline src/main/kotlin/com/sourcegraph/cody/context/EmbeddingStatusView.kt]: Why you only changed one setter and not the other? :)

Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/context/EmbeddingStatusView.kt]: fixed

File: src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentCodebase.kt [hunk 2]
```diff
@@ -31,6 +32,12 @@ class CodyAgentCodebase(private val underlying: CodyAgentServer, private val pro
   fun onFileOpened(project: Project, file: VirtualFile) {
     application.executeOnPooledThread {
       onRepositoryNameChange(RepoUtil.findRepositoryName(project, file))
+
+      val openedFileName = file.name
+      val relativeFilePath: String? = ProjectFileUtils.getRelativePathToProjectRoot(project, file)
```
Comment (pkukielka) [inline src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentCodebase.kt:37]: What will be the result if someone open file from outside the workspace?

Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentCodebase.kt:37]: I do not understand. what do you mean by "from outside the workspace"?

Comment (pkukielka) [reply src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentCodebase.kt:37]: File which is not part of the given workspace, e.g. random file from the disk which you drag to the IDE.

Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentCodebase.kt:37]: image

`my_utils.sh` is a file in `home` that does not belong to the workspace. the tooltip does not appear for it.

**Review by pkukielka — APPROVED** [review]
LGTM

**Pull request merged**